### DISCUSSION TITLE
linux/arm64

### DISCUSSION THREAD

**Discussion by sirhaffy** [lloesche/valheim-server-docker#502] [Q&A]
I am trying to run this on my Raspberry Pi and I get this error:

docker: Error response from daemon: image with reference ghcr.io/lloesche/valheim-server was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64.

I have tried to add "--platform linux/arm64 \" but it does not help for me. Any suggestion?

**Comment by lenir67** [comment]
did you ever find a solution i'm running into the same issue

**Reply by sirhaffy** [reply]
No sorry, I gave up.

**Comment by LastSkywalkerER** [comment]
Ive managed to start this image in docker compose container on ARM (ubuntu-server-22) with help of docker emulator

`docker run --privileged --rm tonistiigi/binfmt --install all`

and additional stripe in docker-compose.yml `platform: linux/amd64`

BUT, i have looping message in logs 
`Mar 21 10:28:16 supervisord: valheim-server DEBUG - [2867] - Waiting for server to listen on UDP query port 2457`
and game cant connect to the server (write "connecting" and then show message "cant connect")

When i did the same on AMD platform (ubuntu-22), i dont remember this message and connection was perfect
I simply rsync entire server from working ubuntu amd environment to arm ubuntu environment, change path for env in docker compose from `$HOME/.../...env` to `...env` (because it cant find it on arm platform) and then run `docker compose up`. After all on arm it run, but havent connection.

**Reply by LastSkywalkerER** [reply]
maybe this will help, judging by the logs inside and outside the container, the server basically does not start and does not listen to any ports.

<img width="972" alt="1" src="https://github.com/lloesche/valheim-server-docker/assets/64728172/d9485913-b8bb-4848-b1d3-b7b35e4a5e28">
<img width="1066" alt="2" src="https://github.com/lloesche/valheim-server-docker/assets/64728172/35b22e8d-52ee-4e93-ac2a-91c77ff0eec6">
<img width="977" alt="3" src="https://github.com/lloesche/valheim-server-docker/assets/64728172/2639314e-34a6-4ba5-865b-2f84d2117a9f">

**Comment by waheedullahkhan001** [comment]
Any progress?
